doc/man3/RAND_add.pod

```diff
@@ -54,6 +57,13 @@ should consider using L<RAND_load_file(3)> instead.
 
 RAND_seed() is equivalent to RAND_add() with B<randomness> set to B<num>.
 
+RAND_keep_random_devices_open() is used to control the resource use of
+the random device seed sources. Some seed sources are able to maintain
+open file descriptors which allow them to operate in the current process's
+chroot(2) jails without the associated device node being available. This
+call must be made before calling chroot(2).
```

Comment:
The documentation appears misleading, because it gives impression that this is something chroot-ing application, and **only** chroot-ing application, would have to do. While current reality is that such application doesn't have to do anything. And it's non-chrooting applications, i.e. overwhelming majority of them, would find themselves in position of deciding whether or not they need to make the call. If intention is to reverse situation in later release, then it should say so. I.e. that the call **will** have to be made before calling chroot, I also have difficulty parsing "are able" in "seed sources are able to maintain open file descriptors". What does "ability" mean in the context? First sentence speaks of "resources", so wouldn't it be more appropriate to say that open file descriptors are case of the said resources...

Reply:
I've reworded it to make the behaviour and usage more explicit.